sbc_compassion/demo/correspondence_template_postman.xml

```diff
@@ -0,0 +1,133 @@
+<odoo>
```

Comment:
maybe remove postman from file name as you define several templates in this file